src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -543,10 +543,22 @@ public void onColonyTick(final IColony colony)
             {
                 respawnInterval = MineColonies.getConfig().getServer().citizenRespawnInterval.get() * TICKS_SECOND;
                 // Make sure the initial citizen contain both genders
+                int femaleCount = 0;
+                // 50-50 chance for first citizen to be male or female.
+                if(colony.getCitizenManager().getCitizens().isEmpty() && random.nextBoolean())
+                {
+                    femaleCount--;
```

Comment:
that seems like a hacky solution, why not make this a bigger if/else and setIsFemale(true); else, the rest of the code